specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-06-01-preview/keys.json

```diff
@@ -589,11 +589,11 @@
       "properties": {
         "timeAfterCreate": {
           "type": "string",
-          "description": "The time duration after key creation to rotate the key. It should be in ISO8601 format. Eg: 'P90D', 'P1Y'."
+          "description": "The time duration after key creation to rotate the key. It only applies to rotate. It should be in ISO8601 format. Eg: 'P90D', 'P1Y'."
```

Comment:
The same needs to be done here as well. I did comment on the other feedback to replace all instances in this PR. ;)

Reply:
Done